using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -311,7 +323,7 @@ NSMutableDictionary *contextData = [NSMutableDictionary dictionary];
 
 ## Configuration keys
 
-To update the SDK configuration programmatically, use the following information to change your Analytics configuration values. For more information, see [Configuration Methods Reference](./).
+To update the SDK configuration programmatically, use the following information to change your Analytics configuration values. For more information, see [Configuration API Reference](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-api-reference).
```

Comment:
Please structure as follows:
"To update the SDK configuration programmatically, use the following information to change your Analytics configuration values:

(code sample)

For more information, see [Configuration API Reference](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-api-reference).

Reply:
Where does that table go? Should it be like following?

"To update the SDK configuration programmatically, use the following information to change your Analytics configuration values:

that key table 

(code sample)

For more information, see Configuration API Reference.

Reply:
> Where does that table go? Should it be like following?
> 
> "To update the SDK configuration programmatically, use the following information to change your Analytics configuration values:
> 
> that key table
> 
> (code sample)
> 
> For more information, see Configuration API Reference.

yep @addb...that's correct.